Controllers/Frontend/Transparent.php

```diff
@@ -7,7 +7,7 @@
 class Shopware_Controllers_Frontend_Transparent extends Shopware_Controllers_Frontend_Payment implements CSRFWhitelistAware
 {
 
-    const ALLOWED_PARAMS = ['MD', 'PaRes', 'payload'];
+    const ALLOWED_PARAMS = ['MD', 'PaRes', 'payload', 'redirectResult'];
```

Comment:
I am not 100% sure how it works in SW5 but if it's a GET request it should forward all the parameters in GET without filtering. If it's a POST it should apply the allowed_params filtering.

Reply:
Gotcha, will send all GET params and filter only POST instead.